trap.go

```diff
@@ -133,6 +120,33 @@ func main() {
 			}
 		}
 	}()
-	log.Println("initialized.")
+	log.Printf("initialized. %s mode\n", conf.RunningMode())
 	wg.Wait()
 }
+
+func checkMackerelConfig(conf *config.Mackerel) (*mackerel.Client, error) {
+	if conf.ApiKey == "" {
```

Comment:
```suggestion
	if conf == nil || conf.ApiKey == "" {
```

そんなことすんなよ…と言われそうですが、mackerel:から一式全部消したらpanicしたので、チェック入れてみます。

Reply:
ナイスキャッチです。